src/Microsoft.Azure.SignalR.Common/Pagination/GroupMemberQueryResultPage.cs

```diff
@@ -34,3 +32,58 @@ public override Response GetRawResponse()
         throw new NotSupportedException();
     }
 }
+
+internal class GroupMemberQueryResultPageConverter : JsonConverter<GroupMemberQueryResultPage>
```

Comment:
why not simply use `[JsonProperty("value")]`/`[JsonProperty("nextlink")]`

Reply:
If we use `JsonProperty("value")`, we need to load the JSON object into memory first, but in this way, we don't need to load the JSON object, just process it streamingly.